AzFilesHybrid/AzFilesHybrid.psm1

```
@@ -219,6 +219,61 @@ function Assert-IsDomainJoined {
     }
 }
 
+function Assert-IsNotAADDS {

```

Better name it Assert-IsAD or Assert-IsNativeAD. It should fail if the storage account is not configured for AD.

---

Resolved